apps/meteor/app/api/server/v1/users.ts

```diff
@@ -1340,6 +1341,9 @@ API.v1.addRoute(
 						user: { status, _id, username, statusText, roles, name },
 						previousStatus: user.status,
 					});
+
+					const calendarService = new CalendarService();
```

Comment:
I'm not sure about creating a service instance only for calling the method. The "Services" are created more to be registered inside the broker, and we are not using services as business logic holders (yet, at least). WDYT?

Reply:
`import { Calendar } from '@rocket.chat/core-services';`